init.lua

```diff
@@ -125,7 +117,7 @@ function stamina.update_saturation(player, level)
 
 	local old = stamina.get_saturation(player)
 
-	if level == old then  -- To suppress HUD update
+	if level == old then -- To suppress HUD update
```

Comment:
This shouldn't have been changed, there should be two spaces before inline comments.

https://dev.minetest.net/Lua_code_style_guidelines#Comments